pages/index.js

```diff
@@ -31,13 +40,21 @@ export default function HomePage() {
     }
     setCurrentCard(library[nextCardIndex]);
   }
-
+  function handleWrongAnswer(damage) {
+    if (userArmor >= damage) {
+      setUserArmor((prevUserArmor) => prevUserArmor - damage);
+    } else {
+      setUserHealth((prevUserHealth) => prevUserHealth - damage);
+    }
+  }
   return (
     <div>
       <Question question={currentCard.question} />
```

Comment:
Maybe depending on what else you want to do, it may make sense or be useful to destructure "currentCard" and then hand down only "question" and "answer" as props

Reply:
thx